### PR TITLE
[UXE-2684] fix: default rule must be the first in the list and has no sort icon

### PR DESCRIPTION
**Pull request by pauloSF0**
## Pull Request
[UXE-2684] 

### What is the new behavior introduced by this PR?
fix: default rule must be the first in the list and has no sort icon
### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.

https://github.com/aziontech/azion-console-kit/assets/110847590/bd34a2ae-dec5-4f83-bf9f-920bbbca9780

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [ ] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

[UXE-2684]: https://aziontech.atlassian.net/browse/UXE-2684?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lucasmendes21 — DISMISSED** [review]

File: src/services/edge-application-rules-engine-services/list-rules-engine-service.js [hunk 1]
```diff
@@ -52,8 +52,14 @@ const adapt = (httpResponse) => {
     }
   })
 
+  const sortParsedRulesEgine = parsedRulesEgine.sort((sortA, sortB) => {
+    if (sortA.name === 'Default Rule') return -1
```
Comment (matheus-sunderhus) [inline src/services/edge-application-rules-engine-services/list-rules-engine-service.js:56]: evitar os números mágicos, colocar -1 e 1 em variáveis com nomes descritivos.

File: src/services/edge-application-rules-engine-services/list-rules-engine-service.js [hunk 2]
```diff
@@ -52,8 +52,14 @@ const adapt = (httpResponse) => {
...
+    if (sortB.name === 'Default Rule') return 1
+    return 0
```
Comment (matheus-sunderhus) [inline src/services/edge-application-rules-engine-services/list-rules-engine-service.js:58]: adicionar uma variável também para o zero , por exemplo:
const keepSameOrder = 0;
const putItAfter = 1;
const putIfBefore = -1;

**Review by lucasmendes21 — APPROVED** [review]

File: src/services/edge-application-rules-engine-services/list-rules-engine-service.js [hunk 3]
```diff
@@ -52,8 +52,18 @@ const adapt = (httpResponse) => {
     }
   })
 
+  const keepSameOrder = 0
+  const putItAfter = 1
+  const putIfBefore = -1
+
+  const sortParsedRulesEgine = parsedRulesEgine.sort((sortA, sortB) => {
```
Comment (matheus-sunderhus) [inline src/services/edge-application-rules-engine-services/list-rules-engine-service.js:59]: sortA, mudar nome para currentItem e sortB para nextItem.
Evitar variáveis com nome do método ou que não passem a ideia do que é o objeto no loop.

**Review by matheus-sunderhus — COMMENTED** [review]
review variable names

**Review by matheus-sunderhus — APPROVED** [review]

**Review by aloisio-m-bastian — APPROVED** [review]

**Pull request merged**